retrofit/src/test/java/retrofit2/CallTest.java

```diff
@@ -978,4 +985,66 @@
     }
     assertThat(writeCount.get()).isEqualTo(1);
   }
+
+  @Test public void queryParamsSkippedIfConvertedToNull() throws IOException, InterruptedException {
```

Comment:
Do these tests belong in `RequestBuilderTest`? They just need to ensure that the request is created correctly. I don't think they care about execution of calls.